geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+GMDIR="/opt/geomesa-hbase_2.11-%%project.version%%"
+
+if [[ ! -d "${GMDIR}" ]]; then
+  echo "Unable to find geomesa directory at ${GMDIR}"
+  exit 1
+fi
+
+echo "Bootstrapping GeoMesa HBase with version %%project.version%% installed at ${GMDIR}"
+
+pip install --upgrade awscli
+
+if [[ ! -d "/opt" ]]; then
+  echo "Unable to find /opt"
+  exit 1
+fi
+chmod a+rwx /opt
+
+ln -s ${GMDIR} /opt/geomesa
+
+cat <<EOF > /etc/profile.d/geomesa.sh
+export GEOMESA_HOME=/opt/geomesa
+export HBASE_HOME=/usr/lib/hbase
+export HADOOP_HOME=/usr/lib/hadoop
+export PATH=\$PATH:\$GEOMESA_HOME/bin
+
+EOF
+
+## Make sure 'hbase' is up first!
+
+ROOTDIR=`cat /usr/lib/hbase/conf/hbase-site.xml 2> /dev/null | tr '\n' ' ' | sed 's/ //g' | grep -o -P "<name>hbase.rootdir</name><value>.+?</value>" | sed 's/<name>hbase.rootdir<\/name><value>//' | sed 's/<\/value>//'`
+while [ -z "$ROOTDIR" ]
+do
+      sleep 2
+      echo Waiting for HBase to be configured.
+      ROOTDIR=`cat /usr/lib/hbase/conf/hbase-site.xml 2> /dev/null | tr '\n' ' ' | sed 's/ //g' | grep -o -P "<name>hbase.rootdir</name><value>.+?</value>" | sed 's/<name>hbase.rootdir<\/name><value>//' | sed 's/<\/value>//'`
+done
+
+# Copy AWS dependencies to geomesa lib dir
+# Make sure HBase is properly initialized before running this
+# script or else the configuration will not be set up
+# properly
+cp /usr/share/aws/emr/emrfs/lib/* /opt/geomesa/lib
+cp /usr/lib/hbase/conf/hbase-site.xml /opt/geomesa/conf/
```

Comment:
Hmm...  the files in /opt/geomesa/conf aren't getting picked up in the classpaths.  I'm tempted to leave it (in case I'm missing a reason for it...)